examples/fabric/v4/cloudRouterConnectivity/routing-protocol-bgp/README.md

```diff
@@ -6,24 +6,24 @@ make a copy of the base folder - examples/routing-protocol-bgp/ and CD into this
 
 ## Define values for the Fabric Cloud Router create
 At minimum, you must set below variables in `terraform.tfvars` file:
-- `equinix_client_id` - Equinix client ID (consumer key), obtained after
-  registering app in the developer platform
-- `equinix_client_secret` - Equinix client secret ID (consumer secret),
-  obtained same way as above
-- `rp_name`- Name of routing Protocol
-- `rp_type`- Type of routing Protocol entity, "BGP"
-- connection_uuid = "d557cb4c-9052-4298-b5ca-8a9ed914cf03"
-  rp_type = "DIRECT"
-  rp_name = "FCR-RP"
-  customer_peer_ipv4 = "192.1.1.2"
-  customer_peer_ipv6 = "192::1:2"
-  customer_asn = "100"
+- `equinix_client_id` - Equinix client ID (consumer key), obtained after registering app in the developer platform
+- `equinix_client_secret` - Equinix client secret ID (consumer secret), obtained same way as above
+- `rp_name` - Name of routing Protocol
+- `rp_type` - Type of routing Protocol entity, "BGP"
+- `connection_uuid` - "d557cb4c-9052-4298-b5ca-8a9ed914cf03"
```

Comment:
Another follow up on this, the README.md should have descriptions of what the values are or what they are coming from. They shouldn't be a value directly. 

I.E. the `rp_name` and `rp_type` above are correct but the second occurrence of them isn't. Additionally, the second occurrence of them should be deleted.

All other values in here that are a value instead of a description should be changed.